plugin/src/main/cpp/register_types.cpp.orig

```diff
@@ -0,0 +1,379 @@
+/**************************************************************************/
```

Comment:
Looks like this `.orig` file accidentally got commited

Reply:
removed that